core-common/src/main/java/org/mapstruct/BeanMapping.java

```diff
@@ -83,4 +83,18 @@
      * @since 1.3
      */
     boolean ignoreByDefault() default false;
+
+    /**
+     * The unmapped source properties that need to be ignored. This should be used when {@link ReportingPolicy#WARN}
+     * or {@link ReportingPolicy#ERROR} is used for {@link Mapper#unmappedSourcePolicy()} or
+     * {@link MapperConfig#unmappedSourcePolicy()}. When properties are set they won't be part of the unmapped source
```

Comment:
Listed properties will be ignored when composing the unmapped source properties report.